<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication0Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication0Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 228, 255);</string>
    </property>
    <widget class="QWidget" name="main_first">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>90</y>
        <width>621</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Desktop/Снимок экрана 2021-11-05 011636.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="setPassword">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>260</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
     border: 2px solid gray;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="setLogin">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>200</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
     border: 2px solid gray;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="question">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>469</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No account? Register now</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Register">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>469</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background_color: rgba(255,255,255,0);
border: none;
color: blue;</string>
      </property>
      <property name="text">
       <string>Register now</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Log_in">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>340</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-style: solid;
border-width: 2px;
border-radius: 10px;
border-color: lightGray;
font: bold 14px;
min-width: 10em;
padding: 6px;

</string>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>361</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Downloads/rounded-in-photoretrica (1).png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>650</y>
        <width>1111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>This program is designed to take into account the availability of places in kindergartens and is not a commercial project.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_inf">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>410</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>110</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main_second">
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>116</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

     border-style: solid;
     border-width: 3px;
     border-radius: 10px;
     border-color: lightGray;
 
font: 14pt &quot;Rockwell&quot;;
     min-width: 4em;
     padding: 3px;

</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>821</width>
        <height>611</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Desktop/Снимок экрана 2021-11-05 011636.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="setPasswordReg">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>290</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
     border: 2px solid gray;
     border-width: 3px;
     border-radius: 10px;
     border-color: lightGray;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="setLoginReg">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
     border: 3px solid gray;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Register_begin">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>440</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

     border-style: solid;
     border-width: 3px;
     border-radius: 10px;
     border-color: lightGray;
   font: 12pt &quot;Rockwell&quot;;
     min-width: 10em;
     padding: 6px;

</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RePassword">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
     border: 2px solid gray;
     border-width: 3px;
     border-radius: 10px;
     border-color: lightGray;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>510</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>555</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>39</y>
        <width>281</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 234);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
font: 16pt &quot;Rockwell&quot;;

     border-style: solid;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
    
     min-width: 6em;
     padding: 3px;


</string>
        </property>
        <property name="text">
         <string>Manage accounts</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
font: 16pt &quot;Rockwell&quot;;

     border-style: solid;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
    
     min-width: 6em;
     padding: 3px;
</string>
        </property>
        <property name="text">
         <string>Manage data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
font: 16pt &quot;Rockwell&quot;;

     border-style: solid;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
    
     min-width: 6em;
     padding: 3px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
font: 16pt &quot;Rockwell&quot;;

     border-style: solid;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
    
     min-width: 6em;
     padding: 3px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>590</y>
          <width>103</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
font: 16pt &quot;Rockwell&quot;;

     border-style: solid;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
    
     min-width: 3em;
     padding: 3px;
</string>
        </property>
        <property name="text">
         <string>Log out</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3"/>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_3">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>39</y>
        <width>1001</width>
        <height>720</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit account:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Back_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>116</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

     border-style: solid;
     border-width: 3px;
     border-radius: 10px;
     border-color: lightGray;
 
font: 14pt &quot;Rockwell&quot;;
     min-width: 4em;
     padding: 3px;

</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="setLogin_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>200</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
     border: 2px solid gray;
     border-width: 2px;
     border-radius: 10px;
     border-color: lightGray;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user_menu">
     <widget class="QStackedWidget" name="stackedWidget_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>39</y>
        <width>281</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 234);</string>
      </property>
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page_7"/>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_5">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>39</y>
        <width>1001</width>
        <height>720</height>
       </rect>
      </property>
      <widget class="QWidget" name="page_8"/>
      <widget class="QWidget" name="page_9"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication0.qrc"/>
 </resources>
 <connections/>
</ui>
